sc2reader/resources.py

```diff
@@ -914,7 +914,7 @@ def __getstate__(self):
 
 
 class Map(Resource):
-    url_template = "http://{0}.depot.battle.net:1119/{1}.s2ma"
+    url_template = "https://{0}-s2-depot.classic.blizzard.com/{1}.s2ma"
```

Comment:
```suggestion
    url_template = "https://{}-s2-depot.classic.blizzard.com/{}.s2ma"
```
The 0 and 1 have not been needed since Python 2.6.

Reply:
I've adjusted all the templates that use numbers.